### ISSUE TITLE
Checkbox Group returns the previous (not current) array of checked boxes

### ISSUE THREAD

**Issue by MiLeung** [react-native-material-design/react-native-material-design#75]
For example if you first check the first box

```
onSelect={(value) => {
            console.log('value', value);
          }}
```

will print out "value []"

And if you uncheck the box "value [0]" will be printed out

**Comment by MiLeung** [comment]
the selected array is updated after the callback is called

**Comment by cldwalker** [comment]
I'm also seeing this bug with 0.3.6 and RN 0.26.0. I don't think I was seeing this bug when using 0.3.3 and RN 0.24.0

**Comment by cldwalker** [comment]
@MiLeung I put up a fix in #76. If you'd like to try the fix, update your package.json: `"react-native-material-design": "git://github.com/cldwalker/react-native-material-design#fix-checkbox-group-on-select"`

**Comment by MiLeung** [comment]
Somewhat unrelated question, but how did you create that url ("git://github.com/cldwalker/react-native-material-design#fix-checkbox-group-on-select") that has your forked repo with the fixes?  I would like to add some additional fixes
